tests/testStreamFramesProcessed.cpp

```diff
@@ -70,10 +70,11 @@ TEST_P(StreamFramesProcessed, VerifyFramesProcessed) {
     sleep(PROCESS_TIME_SECONDS);
 
     // The frames written should be close to sampleRate * PROCESS_TIME_SECONDS
+    const int kDeltaFramesWindowInFrames = 30000;
```

Comment:
Changed this to a const instead of a multiple of the sample rate. Drifting doesn't generally get worse with a higher sample rate